portfolio.css

```diff
@@ -0,0 +1,300 @@
+	display: flex;
+	justify-content: space-between;
+	position: relative;
+	top: 100px;
+}
+
+.about-me-img {
+	height: 200px;
+}
+
+h2 {
+	margin: 20px;
+	text-align: center;
+	font-size: 2rem;
+	font-family: "Courier New";
+}
+
+.projects-anchor {
+	font-size: 2rem;
+	color: black;
+}
+
+.my-projects {
+	margin-top: 120px;
+	margin-bottom: 50px;
+	display: block;
+}
+
+.project-section {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-around;
+	font-family: "Courier New";
+	padding-left: 100px;
+	padding-right: 100px;
+}
+
+.project-div {
+	margin: 20px;
+	width: 35%;
+	font-size: 1rem;
+}
+
+button {
+	font-family: "Courier New";
+	border: 2px solid black;
+	padding: 5px 12px;
+	border-radius: 50%;
+	font-size: 0.8rem;
+	box-shadow: 6px 6px 5px;
+}
+
+button:hover {
+	background-color: yellow;
+}
+
+.project-para {
+	margin-bottom: 15px;
+}
+
+h3 {
+	margin-bottom: 10px;
+}
+
+.sm-footer {
+	padding-top: 50px;
+	padding-bottom: 50px;
+	background-color: #ceced6;
+}
+
+.sm-section {
+	display: flex;
+	justify-content: center;
+	width: 50%;
+	margin-left: auto;
+	margin-right: auto;
+	font-family: "Courier New";
+}
+
+.contact-image {
+	display: block;
+	height: 300px;
+	border-radius: 55%;
+	margin-right: 30px;
+}
+
+.contact-div {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	margin-top: auto;
+	margin-bottom: auto;
+	font-size: 1rem;
+}
+
+.sm-image {
+	display: block;
+	height: 100px;
+	transition: transform .2s;
+}
+
+.sm-image:hover {
+	transform: scale(0.7);
+}
+
+.sm-text {
+	font-size: 1rem;
+	padding-top: 42px;
+}
+
+.sm-div {
+	display: flex;
+	justify-content: space-between;
+	height: 100px;
+}
+
+.email {
+	margin-top: 10px;
+	margin-left: 30px;
+}
+
+@media only screen and (max-width: 1000px) {
+	.info-about-me {
+		width: 80%;
+	}
+	
+	.project-div {
+		width: 80%;
+	}
+}
+
+@media only screen and (max-width: 710px) {
+	.about-me-img {
+		height: 100px;
+		position: relative;
+		bottom: 50px;
+	}
+	
+	.navbar {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+	}
+	
+	.nav-anchor {
+		margin-right: 0;
+	}
+	
+	.project-section {
+		padding-left: 50px;
+		padding-right: 50px;
+	}
+}
+
+@media only screen and (max-width: 650px) {
+	.sm-section {
+		display: flex;
+		flex-direction: column;
+		margin-left: auto;
+		margin-right: auto;
+	}
+	
+	.nav-anchor {
+		margin-right: 0;
+	}
+}
+
+@media only screen and (max-width: 550px) {
+	.nav-anchor2 {
+		font-size: 1rem;
+		padding: 10px;
+	}
+	
+	.nav-anchor {
+		font-size: 1rem;
+		padding: 10px;
+		margin-right: 0;
+	}
+}
+
+@media only screen and (max-width: 500px) {
+	.project-div {
+		width: 90%;
+	}
+}
+
+
+@media only screen and (max-width: 450px) {
+	.sm-section {
+		margin-left: 85px;
+	}
+	
+	.project-section {
+		padding-left: 20px;
+		padding-right: 20px;
+	}
+}
+
+@media only screen and (max-width: 410px) {
+	.img-container {
+		width: 90%;
+	}
+}
+
+@media only screen and (max-width: 390px) {
```

Comment:
There's a lot of different max-width values here. Is it possible to use a single value or does each one need a specific one ?

Also, it's tricky to follow the media queries, it would be easier to follow if the media query for an element is next to the original element css